apiary-metastore-metrics/src/main/java/com/expediagroup/apiary/extensions/metastore/metrics/CodahaleMetrics.java

```diff
@@ -303,6 +303,18 @@ public Long decrementCounter(String name, long decrement) {
     }
   }
 
+  @Override
+  public void removeGauge(String name) {
+
+    if (!metricRegistry.getGauges().containsKey(name)) {
+      LOGGER.warn("Cannot remove gauge[" + name + "] as it is not currently registered.");
+      return;
+    }
+
+    gauges.remove(name);
+    metricRegistry.remove(name);
+  }
+
```

Comment:
`org.apache.hadoop.hive.common.metrics.common.Metrics` impl requires `removeGauge` to implemented now.